### PR TITLE
module 2 task

### PR DESCRIPTION
**Pull request by ghost**
@szikszail please review my tests for second task. Thank you

### CODE REVIEW

File: module-2/test/calc.spec.js [hunk 1]
```diff
@@ -19,5 +20,137 @@ describe.only('calc', () => {
...
+        });
+        const testData = [
+            1000, 50, 30,
+            0, 222222
+        ];
+        testData.forEach(n => {
+            it(`sqrt function should work correctly for ${n}`, () => {
+                const calcClass = calc(n);
+                const result = Math.sqrt(n)
+                return expect(calcClass.sqrt(n).v).to.equal(result);
```
Comment (szikszail) [inline module-2/test/calc.spec.js]: `sqrt` does not accept any parameter.

**Review by szikszail — CHANGES_REQUESTED** [review]
Good tests!

File: module-2/test/calc.spec.js (hunk 1 above)
Comment (ghost) [reply module-2/test/calc.spec.js]: Changed

**Review by szikszail — APPROVED** [review]

**Comment by szikszail** [comment]
Nice job

**Pull request closed without merging**